### DISCUSSION TITLE
React-style framework

### DISCUSSION THREAD

**Discussion by globophobe** [widgetti/solara#150] [General]
Solara's description as a react-style framework confused me. To me, that meant that, like Plotly Dash, the components available were React components. I was pleasantly surprised that many of the components in the examples are ipyvuetify. As a result, I realized that Solara can use Vue components.

I understand the intent, but I wonder if other people are similarly confused by the React description.

**Comment by maartenbreddels** [comment]
Hi Alex,

Thank you for your feedback. Solara's main focus is to provide a Python API, and make components on the Python side using a React-like API. Making custom frontend components is an advanced use case, which we do not advertise that much yet.
Does your confusion stem from dash advertising that it is built on top of React?
We plan to support both Vue and React, and might even do a Svelte example, since we are mostly frontend tooling agnostic.
Where in the docs would you expect to read about this topic?

Regards,

Maarten

**Comment by globophobe** [comment]
> Does your confusion stem from dash advertising that it is built on top of React?

Yes, initially I read `React-style` in the description, and believed Solara to be similar to Dash, Pynecone, etc.

<img src="https://github.com/widgetti/solara/assets/891192/40bae84f-bded-49e3-a52f-b663139aa589" width="30%" />

A few weeks later, I looked at the docs, ran some examples, and realized I was wrong. Currently, I'm preparing a talk with the ipvuetify components. I couldn't be happier that I can embed my python demos in the presentation :tada:
